articles/metadata/apiv2.md

```diff
@@ -91,7 +96,7 @@ The `PATCH` call to the API would therefore be:
 }
 ```
 
-The user's profile now appears as follows:
+The user's profile will now appears as follows:
```

Comment:
There was a small typo in the original: `will now appear as follows` instead of "will now appears". Maybe you could fix that in this PR as well? :)